backends-common/cassandra/pom.xml

```diff
@@ -48,8 +48,8 @@
                 exclude this dependency explicitly: -->
             <exclusions>
                 <exclusion>
-                    <groupId>io.netty</groupId>
                     <artifactId>*</artifactId>
+                    <groupId>io.netty</groupId>
```

Comment:
The plugin preserves groupId/artifactId in dependency but apparently not in exclusion.

I don't think I can do much... If I fix it, it will reappear next time someone run the sort order dependency. I would argue "automatic sorting" outweigh this little drawback.